.github/workflows/product_translations/products_pot

```diff
@@ -51,7 +51,7 @@ class POFile {
    */
   timestamp() {
     const date = new Date();
-    return date.getUTCFullYear() + "-" + date.getUTCMonth() + "-" +
+    return date.getUTCFullYear() + "-" + (date.getUTCMonth() + 1) + "-" +
```

Comment:
A fix for the previous PR, it turned out that the month (and only the month!!) starts from zero.